src/test/fuzz/eval_script.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2009-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <script/interpreter.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+
+#include <limits>
+
+void test_one_input(const std::vector<uint8_t>& buffer)
+{
+    if (buffer.size() <= sizeof(unsigned int)) {
+        return;
+    }
+    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
```

Comment:
style-nit: Looks like this creates another copy of the buffer.

Could be solved by making `test_one_input` pass either a reference of a vector or a fuzzed data provider at compile time (with a compile flag)?

Feel free to ignore.